platform/debian/kanidmd/rules

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/make -f
+# See debhelper(7) (uncomment to enable)
+
+# be REALLY noisy
+export DH_VERBOSE=1
+export DH_OPTIONS=-v
+
+export DEB_BUILD_OPTIONS=noautodbgsym
+
+export DEB_BUILD_OPTIONS=noautodbgsym
+PACKAGE=kanidmd
+PKGDIR=debian/${PACKAGE}
+BINDIR=${PKGDIR}/usr/sbin/
+SHARED_DIR=${PKGDIR}/usr/share/${PACKAGE}
+
+%:
+	dh $@
+
+override_dh_auto_clean:
+# cargo clean
+
+override_dh_autoreconf:
+
+override_dh_auto_build:
+	./platform/debian/fix_wasm_opt.sh
+	cd kanidmd_web_ui && ./build_wasm.sh
+	dh_auto_build -- release/kanidmd
```

Comment:
Again is this using the makefile above? If it's for debian, it'll need a platform profile and release opts, ?

Reply:
yep, will continue that discussion above